src/misc/00_intro.rst

```diff
@@ -6,6 +6,9 @@ Glossary
 
 .. glossary::
 
+  ENV  
```

Comment:
Would just spell this out

```suggestion
  Environment
```

Reply:
@jakirkham The suggested change has been made in the below commit